css/style_float.css

```diff
@@ -0,0 +1,137 @@
+@import "fonts.css";
+
+html {
+    box-sizing: border-box;
+    background: url("../image/background/backgroundImage.png") no-repeat center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+*,
+*:before,
+*:after {
+    box-sizing: inherit;
+}
+
+/*-----------split by block and style block-------------*/
+.content {
+    max-width: 600px;
+    margin-right: auto;
+    margin-left: auto;
+    margin-top: 10%;
+    box-shadow: 10px -10px 50px rgba(0, 0, 0, 0.8),
+    10px 10px 50px rgba(0, 0, 0, 0.8),
+    -10px -10px 50px rgba(0, 0, 0, 0.8),
+    -10px 10px 50px rgba(0, 0, 0, 0.8);
```

Comment:
It would be better to use smaller values for opacity in color for box-shadow, it'll make shadows softer.